newIDE/app/src/stories/componentStories/Credits/CreditsPackagesDialog.stories.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+import * as React from 'react';
+
+import muiDecorator from '../../ThemeDecorator';
+import paperDecorator from '../../PaperDecorator';
+import CreditsPackagesDialog from '../../../Credits/CreditsPackagesDialog';
+import { CreditsPackageStoreContext } from '../../../AssetStore/CreditsPackages/CreditsPackageStoreContext';
+
+export default {
+  title: 'Credits/CreditsPackagesDialog',
+  component: CreditsPackagesDialog,
+  decorators: [paperDecorator, muiDecorator],
+};
+
+export const Loading = () => {
+  return (
+    <CreditsPackageStoreContext.Provider
+      value={{
+        fetchCreditsPackages: () => {},
+        creditsPackageListingDatas: null,
+        error: null,
+      }}
+    >
+      <CreditsPackagesDialog onClose={() => {}} />
+    </CreditsPackageStoreContext.Provider>
+  );
+};
+
+export const Loaded = () => {
+  return (
+    <CreditsPackageStoreContext.Provider
+      value={{
+        fetchCreditsPackages: () => {},
+        creditsPackageListingDatas: [
```

Comment:
Data to put in the fixtures?

Reply:
I don't think this will be used anywhere else so for the moment leaving it here